### DISCUSSION TITLE
Question about downloading past broadcasts off Picarto

### DISCUSSION THREAD

**Discussion by FallenMans** [streamlink/streamlink#4184] [Q&A] [answered]
I am attempting to download VODs of past streams on Picarto to watch at a later date before they get deleted and shoving the VOD link into streamlink doesn't work but going and taking the direct m3u8 index link does, however, is it possible to download one of these vods/active streams without the audio? I'm only really interested in the video side of things

**Comment by bastimeyer** [comment, marked as answer]
> m3u8
> download one of these vods/active streams without the audio?

HLS streams (MPEG-TS) usually carry both video and audio streams, so you can't just download the video data. You'll have to re-mux the output file afterwards and remove the streams which you don't want (audio).

```sh
streamlink --output "recording.ts" STREAM QUALITY
ffmpeg -i "recording.ts" -c copy -an "output.mp4"
```

`-c copy` means copy all streams and `-an` means no audio streams.

**Reply by mkbloke** [reply]
#4132
#4140
